input/fsh/SD_RadiotherapyTreatmentPhase.fsh

```diff
@@ -12,21 +12,24 @@ Please note that while these codes may imply conformance to the profiles that us
 2. `Procedure.code`, `Observation.code`, and `Group.code` elements require a [CodeableConcept data type](http://www.hl7.org/fhir/datatypes.html#CodeableConcept), which can be satisfied with a code from this code system. In contrast, a profile is identified with a [canonical URL](http://www.hl7.org/fhir/structuredefinition-definitions.html#StructureDefinition.url), which is represented by a [URI data type](http://www.hl7.org/fhir/datatypes.html#uri). In other words, a profile's canonical URL belongs in the `meta.profile` element, whereas the codes in this code system belong in `Resource.code` elements.
 3. The same `code` can correspond to more than one profile (although not the case here)."
 
-* #codexrt-teleradiotherapy-treatment-phase "Teleradiotherapy Treatment Phase Resource" "Identifies a Procedure resource that describes delivery of teleradiotherapy (external beam radiation) and conforms to the TeleradiotherapyTreatmentPhase profile." // SNOMED Code Requested USCRS-33293
-* #codexrt-brachytherapy-treatment-phase "Brachytherapy Treatment Phase Resource" "Identifies a Procedure resource that describes delivery of brachytherapy (internal or surface radiation) and conforms to the BrachytherapyTreatmentPhase profile." // SNOMED Code Requested USCRS-33294
+* #codexrt-teleradiotherapy-treatment-phase "Teleradiotherapy Treatment Phase Resource" "Identifies a Procedure resource that describes delivery of teleradiotherapy (external beam radiation) and conforms to the TeleradiotherapyTreatedPhase profile." // SNOMED Code Requested USCRS-33293
+* #codexrt-brachytherapy-treatment-phase "Brachytherapy Treatment Phase Resource" "Identifies a Procedure resource that describes delivery of brachytherapy (internal or surface radiation) and conforms to the BrachytherapyTreatedPhase profile." // SNOMED Code Requested USCRS-33294
```

Comment:
USCRS-33293 and 33294 have been rejected/withdrawn.
The only code currently requested is RadiotherapyTreatmentPhase USCRS-33527 (parent of brachy and teleradio treatment phases, should we request them).

Reply:
Removed the complete outdated comment section. The USCRS-33527 is properly contained as requested code in DEF_CodeSystems.fsh.